src/OrchardCore/OrchardCore.Data.YesSql/Migration/DataMigrationManager.cs

```diff
@@ -235,7 +237,7 @@ private async Task UpdateAsync(string featureId)
                 }
                 catch (Exception ex)
                 {
-                    _logger.LogError(ex, "Error while running migration version {Version} for '{FeatureName}'.", current, featureId);
+                    _logger.LogError(ex, "Error while running migration '{name}' version {Version} for '{FeatureName}'.", migration.GetType().FullName, current, featureId);
```

Comment:
May as well change `{name}` to `{MigrationName}` for consistency.

Reply:
Done